.gitlab-ci.yml

```diff
@@ -36,10 +36,16 @@ test:
     - pip install pytest certifi
     - if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - python3 setup.py install
+    - curl https://d3gcli72yxqn2z.cloudfront.net/connect_latest/v4/bin/ibm-aspera-connect_4.1.3.93_linux.tar.gz -o ibm-aspera-connect.tar.gz
+    - tar xzvf ibm-aspera-connect.tar.gz
+    - ./ibm-aspera-connect_*_linux.sh
+    - sudo ln -s ~/.aspera/connect/bin/ascp /usr/bin
```

Comment:
If I remember correctly, I was not able to use **sudo** in gitlab-ci for any of the commands, it always gives an error. You might want to check this

Reply:
Removed sudo